src/styles/brackets_fonts.less

```diff
@@ -27,57 +27,53 @@
 @font-weight-light: 200;
 
 
-/* SourceSansPro Regular */
+/* @sansFontFamily Regular */
```

Comment:
curious why these would be kept around if the intention is to no longer use them.